projects.go

```diff
@@ -168,7 +168,9 @@ type ListProjectsOptions struct {
 	Search     *string               `url:"search,omitempty" json:"search,omitempty"`
 	Simple     *bool                 `url:"simple,omitempty" json:"simple,omitempty"`
 	Visibility *VisibilityLevelValue `url:"visibility,omitempty" json:"visibility,omitempty"`
-	Statistics *bool                 `url:"statistics,omitempty" json:"statistics,omitempty"`
```

Comment:
Looking at the docs, I notice that `Statistics` is still a valid option. So could you please add back `Statistics to the bottom of this list? And to satisfy my OCD, could you also relocate `Visibility` to be the last entry? Thx!

```
...
Owned
Membership
Starred
Statistics
Visibility
```